mypy/semanal.py

```diff
@@ -2356,6 +2357,64 @@ def is_classvar(self, typ: Type) -> bool:
     def fail_invalid_classvar(self, context: Context) -> None:
         self.fail('ClassVar can only be used for assignments in class body', context)
 
+    def process_module_assignment(self, s: AssignmentStmt) -> None:
```

Comment:
Yes, good point. This was to present a cleaner API (pass in only the `AssignmentStmt`), but given there's only one call site and will never be more, that's not very useful. Consolidated.